Formula/git.rb

```diff
@@ -149,7 +149,7 @@ def install
     chmod 0644, Dir["#{share}/doc/git-doc/**/*.{html,txt}"]
     chmod 0755, Dir["#{share}/doc/git-doc/{RelNotes,howto,technical}"]
 
-    # git-send-email needs Net::SMTP::SSL
+    # git-send-email needs Net::SMTP::SSL or Net::SMTP >= 2.34
     resource("Net::SMTP::SSL").stage do
       (share/"perl5").install "lib/Net"
     end
```

Comment:
Do we still want to install this on Big Sur or newer?

Reply:
Right now, it is just an extraneous package on macOS 11+ as version check means `Net::SMTP::SSL` should never be used:
```perl
my $use_net_smtp_ssl = version->parse($Net::SMTP::VERSION) < version->parse("2.34");
```

We could wrap it with `if MacOS.version <= :catalina`, though I am not sure if someone on an older Big Sur (e.g. 11.0, 11.1, ...) would see an issue as I recall some Perl updates were done in later versions of Big Sur.

Reply:
Hmm ok let's keep it then